### PR TITLE
Add es6 versions of mythical creatures, wizarding objects, wizarding …

### PR DESCRIPTION
**Pull request by Kalikoze**
…arrays, and update READMEs.

### CODE REVIEW

File: es6-mythical-creatures/test/fairy-test.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+var assert = require('chai').assert;
+var Fairy = require('../exercises/fairy');
```
Comment (damwhit) [inline es6-mythical-creatures/test/fairy-test.js]: Looks like this file is still es5

File: es6-mythical-creatures/test/ogre-test.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+var assert = require('chai').assert;
+var Ogre = require('../exercises/ogre');
+var Human = require('../exercises/human');
```
Comment (damwhit) [inline es6-mythical-creatures/test/ogre-test.js]: same with this one

File: es6-mythical-creatures/test/sphinx-test.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
+var assert = require('chai').assert;
+var Sphinx = require('../exercises/sphinx');
+
+describe('Sphinx', function () {
```
Comment (damwhit) [inline es6-mythical-creatures/test/sphinx-test.js]: and here

File: es6-mythical-creatures/test/fairy-test.js (hunk 1 above)
Comment (Kalikoze) [reply es6-mythical-creatures/test/fairy-test.js]: Good catch, I'll get that fixed.  Thanks David! 👍

**Pull request merged**